src/App.js

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from "react";
+import EditorCards from "./Components/EditorCards";
+import CreateTask from "./Components/CreateTask";
+import "bootstrap/dist/css/bootstrap.min.css";
+import JournalistCards from "./Components/JournalistCards";
+import Tasks from "./Components/Tasks";
+import { BrowserRouter, Route } from "react-router-dom";
+import { Routes } from "react-router-dom";
+import { NavigationBar } from "./Components/NavigationBar";
+import LoginOut from "./Components/LoginOut";
+import Notifications from "./Components/Notifications";
+import Accounts from "./Components/Account";
+import Parse from "parse";
+
+
+
+function App() {
+  //Following text is presented on welcome page
+  const [isLoggedIn, setIsLoggedIn] = useState(false);
+
+  Parse.initialize(
+    "oHPQf2X0grpeXteCxywdMAvZ6BaV3lj1o3SUJ4pJ",
+    "wE0CWYnPVC86eVLQAcvOUm37BFhqW6ZBZTcdCL5B",
+    "7Tqwx4xkFQ0tZDsoQrsI2nxJdlZxXaSev96e5Dtw"
+  );
+
+  Parse.serverURL = "https://parseapi.back4app.com/";
```

Comment:
Consider making an api.js file where you put all api stuff together in one file.
Makes making changes much easier.

Reply:
Solved this by creating a new .env file and added it to git ignore